src/less/index.css

```diff
@@ -194,9 +197,44 @@
 .step-label {
   font-family: moon-regular;
 }
+.images-container {
+  display: inline-flex;
+}
+@keyframes left-header {
+  0% {
+    left: -800px;
+  }
+  100% {
+    left: 0;
+  }
+}
+@keyframes right-header {
+  0% {
+    left: 1000px;
```

Comment:
Are you sure that it will look good on all sizes of browser ? I think that we should use percentages here :/